Kinvey-Xamarin-Android/Auth/UserExtension.cs

```diff
@@ -14,7 +14,7 @@
 using Kinvey;
 using Android.Content;
 
-namespace KinveyXamarinAndroid
+namespace Kinvey
```

Comment:
@heyzooi  You mean that we need to use everywhere "namespace Kinvey", don't you?

Reply:
@romanogolikhin precisely, everywhere but tests and sample projects

Reply:
@heyzooi 
According to it why did you comment this code? :=)

Instead of the "namespace KinveyXamarinAndroid" 
the "namespace Kinvey" is being used.

The "Kinvey-Xamarin-Android" is neither test project nor sample.
All should be ok or I am wrong?

Reply:
@heyzooi @vinaygahlawat 
I had a look at all files in solution. I didn't find KinveyXamarinAndroid and KinveyXamarinIOS namespaces. All code related to Kinvey library has 'Kinvey' namespace. Only tests and app samples have separate namespaces.

But there exists the following projects:
'Kinvey.Utils' and 'Kinvey-Utils(Portable)'.
Their code files have 'KinveyUtils' namespace. Is it correct?

And it seems to me I found obsolete file with 'Kinvey.Net' namespace.
https://github.com/Kinvey/dotnet-sdk/blob/feature/MLIBZ-881-Provide_sample_Xamarin_Forms_app/Kinvey.Net/MyClass.cs
 I think we can remove it.  Am I right?

Reply:
yes, for sure you can remove this class!
In fact, the entire `Kinvey-Utils` project can be removed since only have 2 classes and they were moved to the `Shared` project